src/efi.rs

```diff
@@ -580,6 +609,18 @@ fn find_file_recursive<P: AsRef<Path>>(dir: P, target_file: &str) -> Result<Vec<
     Ok(result)
 }
 
+fn copy_dir(src: &Path, dst: &Path) -> Result<()> {
+    let r = std::process::Command::new("cp")
```

Comment:
Let's find a more "Rust" way to do that.

Reply:
Looks like there isn't anything readily available in the standard library. I found https://stackoverflow.com/questions/26958489/how-to-copy-a-folder-recursively-in-rust as a starting point but it does not preserve access times, etc. like `cp -a` does.

Reply:
It might be reasonable to leave this as is as the code in `cp` is battle tested and very unlikely to fail.

Reply:
See Colin's suggestion https://github.com/coreos/bootupd/pull/669#discussion_r1646764899 to do `cp -a`, just curious how do you check it does not preserve access times?